### PR TITLE
Restful Protocol Fixes: Electricity

### PR DESCRIPTION
**Pull request by shahardo**
#558

### CODE REVIEW

File: negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php [hunk 1]
```diff
@@ -204,6 +209,96 @@ protected function queryForSummary($query) {
...
+    return $result;
+    return array(
+      'min' => $min,
+      'max' => $max,
+    );
+  }
+  protected function calcMinMaxTimestamp() {
+    // Load query and apply filter options.
+    $query = parent::getQuery();
+//    $this->queryForListFilter($query);
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php]: remove

File: negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php [hunk 2]
```diff
@@ -204,6 +209,96 @@ protected function queryForSummary($query) {
...
+
+    // ************* queryForListFilter BEGIN
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php]: move to own method

File: negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php [hunk 3]
```diff
@@ -314,9 +411,12 @@ protected function prepareSummary() {
     // Fill the total section to be output by the formatter.
     $summary['type'] = $result_type;
     $summary['values'] = $total;
+//    $summary['min_max_timestamp'] = $min_max_ts;
```
Comment (amitaibu) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/electricity/1.0/NegawattElectricityResource.class.php]: remove.

**Comment by shahardo** [comment]
Merged

**Pull request closed without merging**